pkg/operator/ceph/object/config.go

```diff
@@ -50,6 +51,9 @@ func (c *clusterConfig) portString() string {
 
 	port := c.store.Spec.Gateway.Port
 	if port != 0 {
```

Comment:
Could port selection here and for the liveness probe share the helper method `generateLiveProbePort()`? Seems like both are computing the same thing separately, or is the difference needed?